### PR TITLE
Release to jenkins-x-versions at end of pipeline

### PR DESCRIPTION
**Pull request by daveconde**
Fixes: #82 

- See [Version Stream](https://jenkins-x.io/architecture/version-stream/#creating-pull-requests) for details

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -77,6 +77,7 @@ pipeline {
 
             // Run updatebot to update other repos
             sh './scripts/updatebot.sh'
+            sh 'jx step create version pr -n $GITHUB_ORG/$APP_NAME -v $(cat VERSION)'
```
Comment (hferentschik) [inline Jenkinsfile:82]: So what exactly does that do?

**Review by hferentschik — COMMENTED** [review]
Generally the same question/comment as on PR #78

File: Jenkinsfile (hunk 1 above)
Comment (hferentschik) [reply Jenkinsfile:82]: I guess what I am asking here is whether one really wants to have each merge to master being merged directly into the version repo. Does that then really solve the version incompatibilities issues.

I guess if the jenkins-x-versions repo would indeed then run the BDD tests and verify that the upgraded version does not break anything, one would indeed get additional level security. 

At the same time, I am still not convinced that every merge to master should indeed do the release. Maybe I am stuck in some old type of thinking.

**Comment by hferentschik** [comment]
BTW, seems there is a conflict which needs resolving.

File: Jenkinsfile (hunk 1 above)
Comment (daveconde) [reply Jenkinsfile:82]: What should happen here is that a PR with the new chart versions will get opened against https://github.com/jenkins-x/jenkins-x-versions. That PR should kick off a BDD test run to certify that the chart is working as expected before the release version is merged to the jenkins-x-versions branch if the BDD tests pass.

I'm currently working on the BDD test side to be able to run the BDD tests against apps.

**Comment by hferentschik** [comment]
@pmuir that was a bit early. This pull request needed a rebase and sorting out of a pull request. Now it won't work

**Pull request merged**